tox.ini

```diff
@@ -66,28 +66,45 @@ commands =
 
 [testenv:cpython-apidocs]
 description = Build CPython API documentation
-
-extras = rst
-
+deps = 
+    pytest
 commands =
     sh -c "if [ ! -d {toxworkdir}/cpython ]; then \
         git clone --depth 1 https://github.com/python/cpython.git {toxworkdir}/cpython; \
         fi"
     sh -c "cd {toxworkdir}/cpython && git pull"
     touch {toxworkdir}/cpython/Lib/__init__.py
     rm -rf {toxworkdir}/cpython-output
-    pydoctor \
+    - pydoctor \
```

Comment:
I don't remember what this does...but I hope that if pydoctor fails here, the whole tox env run also fails.

Reply:
Not it doesn't because pydoctor always returns a non-zero status code when building standard libraty docs. The actual checks are in the python code.

Reply:
ok... I think that twisted system tests were added with the goal at reaching a point at which when running pydoctor on twisted no warnings or errors are generated...and in this way we can better check for any regressions.

if  you are happy with the current tests, I am also happy :)

Reply:
Yes it’s still working like that for twisted, it just fails for the standard library and python-igraph. But that’s ok since we have tests.